misc_docs/syntax/operator_ref_assignment.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: "ref-assignment"
+keywords: ["ref", "assign"]
+name: ":="
+summary: "This is the `ref assignment` operator."
+category: "operators"
+---
+
+This operator is syntactic sugar for assigning to the `contents` value of a `ref`
+
```

Comment:
actually no sugar provided for `ref`, both `ref` and `:=` are functions in pervasives module

Reply:
Gotcha, I was essentially quoting from pages/docs/manual/latest/mutation.mdx:

```
Assign a new value to `myValue` like so:

<CodeTab labels={["ReScript", "JS Output"]}>

\```res example
myValue.contents = 6
\```
\```js
myValue.contents = 6;
\```

</CodeTab>

We provide a syntax sugar for this:

<CodeTab labels={["ReScript", "JS Output"]}>

\```res example
myValue := 6
\```
\```js
myValue.contents = 6;
\```

</CodeTab>
```

Should I re-work this language as well?

Reply:
I added some comments. I think it's not important how this is implemented, we should just describe what it does and how it is useful.